cmd/webhook/webhook.go

```diff
@@ -24,6 +24,7 @@ import (
 var scheme = runtime.NewScheme()
 var codecs = serializer.NewCodecFactory(scheme)
 var resourceBaseName = "nokia.k8s.io"
+var ps_path string
```

Comment:
It's not mentioned in the contribution guide but camel case should be used in variable names. That seems to be current practice so let's follow it. I would also use more descriptive name here so it's clear what the purpose of the variable is.

Reply:
Sorry! I forget that... But it is already corrected.

Reply:
Great !